### PR TITLE
header implementation without menu

### PR DESCRIPTION
**Pull request by Dustikdev**

### CODE REVIEW

**Comment by SimonShvets** [comment]
@Dustikdev  Hi mate! Let's fix this warnings:
image

File: src/_functions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿@function calculate-vw($value) {
+    @return $value * 100 / 1440 * 1vw;
+  }
```
Comment (SimonShvets) [inline src/_functions.scss:3]: I'm not appreciate the flow with _files on source directory, by my opinion these kinds of file should be located in shared components. @Negodyaj what you think?

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+  return (
+    <header>
+      <div className="left-buttons">
+        <img src={burgerMenu} alt="burger-menu" className="mobile-only" />
+        <div className="left-button desktop-only">Shop</div>
+        <div className="left-button desktop-only">Contact</div>
+      </div>
+      <div className="right-buttons">
+        <img src={shopBag} alt="shop-bag" className="mobile-only" />
+        <div className="sign-in-button desktop-only">Sign In</div>
```
Comment (SimonShvets) [inline src/components/Header/Header.tsx:16]: These 'Sign In,' 'Shop,' 'Contact,' etc., should be actual links that lead users to the respective pages.

**Comment by Negodyaj** [comment]
> @Dustikdev Hi mate! Let's fix this warnings: image

тут проще .gitatributes добавить

File: src/_functions.scss (hunk 1 above)
Comment (Negodyaj) [reply src/_functions.scss:3]: да, можете создать папку shared, и кидать туда стили, сервисы, и всю остальную хрень, которая, так или иначе, появится
src обычно держат чистеньким)

**Review by SimonShvets — CHANGES_REQUESTED** [review]
left comments

**Pull request closed without merging**